src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -112,6 +112,10 @@ pub struct RawSourceCreationConfig {
     pub storage_metadata: CollectionMetadata,
     /// The upper frontier this source should resume ingestion at
     pub resume_upper: Antichain<mz_repr::Timestamp>,
+    /// The upper frontier this source should resume ingestion at in the source time domain.
+    /// Since every source has a different timestamp type we carry this frontier in an encoded Row
+    /// form which will get decoded once we reach the connection specialized functions.
+    pub source_resume_upper: Vec<Row>,
```

Comment:
can you explain why its a vector here (that its going to decoded into an antichain)